### PR TITLE
Fixing bug introduced by switch to miscUtils.replaceTextInFile

### PR DESCRIPTION
**Pull request by wmathurin**
Also calling install.sh (the android repo now does npm install in its install.sh, see https://github.com/forcedotcom/SalesforceMobileSDK-Android/pull/1164)

### CODE REVIEW

File: node/test_force.js [hunk 1]
```diff
@@ -270,26 +274,24 @@ function createCompileApp(tmpDir, appType, os) {
 //
 function updatePluginRepo(tmpDir, pluginRepoDir, branch, os) {
     log('Updating cordova plugin at ' + branch, COLOR.green);
-    shelljs.pushd(pluginRepoDir);
-    runProcessThrowError(path.join('tools', 'update.sh') + ' -b ' + branch + ' -o ' + os);
```
Comment (wmathurin) [inline node/test_force.js]: runProcessThrowError can take care of the pushd, popd if passed a directory as second argument

File: node/test_force.js [hunk 2]
```diff
@@ -270,26 +274,24 @@ function createCompileApp(tmpDir, appType, os) {
...
-    shelljs.popd();
+    runProcessThrowError(path.join('tools', 'update.sh') + ' -b ' + branch + ' -o ' + os, pluginRepoDir);
 }
 
 //
 // Update forceios/forcedroid script to use local plugin repo
 //
 function editForceScriptToUseLocalPluginRepo(tmpDir, os) {
     log('Editing  ' + forcePackageNameForOs(os) + '.js to use local cordova plugin', COLOR.green);
-    miscUtils.replaceTextInFile(path.join(tmpDir, 'node_modules', forcePackageNameForOs(os), 'node', forcePackageNameForOs(os) + '.js'), 'cordova plugin add .*', '\'cordova plugin add ../SalesforceMobileSDK-CordovaPlugin\'');
```
Comment (wmathurin) [inline node/test_force.js]: if we pass the regexp in a string, the replace will treat it as a string litteral

**Comment by wmathurin** [comment]
NB: I just ran test_force.js to test generating and compiling app for ios and android.
They all generate (native/native_swift/hybrid_local/hybrid_remote/react_native) but generated hybrid app don't compile for Android.

**Comment by bhariharan** [comment]
👍

**Pull request merged**